src/public/templates/setup.html

```diff
@@ -217,25 +226,33 @@ <h2 class="long" data-loc="vino.setup.screen.pc.h2"></h2>
     </a>
     <div class="about-text l-stick-scroll">
         <h1>What is Nintendo TVii?</h1>
-        <p>Nintendo TVii is an platform that allows you to find and search for TV shows, movies and actors in Streaming
+        <p>Nintendo TVii is an platform that allows you to find and search for TV shows, movies and actors in
+            Streaming
             platforms and Live TV.
             Along with that, it also offers a diverse variety of social features, such as TV Tag.
         </p>
         <br>
         <h1>What is TV Tag?</h1>
-        <p> While you are watching a new episode of a live TV show, the <b>TV Tag</b> feature (if supported) will allow
+        <p> While you are watching a new episode of a live TV show, the <b>TV Tag</b> feature (if supported) will
+            allow
             you to comment on live TV moments with other people as they
-            happen, with a short caption and an attached image/poll/etc. that makes watching TV entertaining and social.
-            Along with commenting, you can also doodle on this attached image, and the comments you do are crossposted
+            happen, with a short caption and an attached image/poll/etc. that makes watching TV entertaining and
+            social.
+            Along with commenting, you can also doodle on this attached image, and the comments you do are
+            crossposted
             to other social media such as
             Miiverse, Bluesky and Tumblr (if set before)</p>
         <br>
         <h1>What can i do with Nintendo TVii?</h1>
         <p>
-            Nintendo TVii offers you the option to see your Favorites, TV shows airing live, or releasing new episodes soon on Streaming platforms,
-            Movies also shows you the currently airing movies live, or the popular/trending ones on Streaming platforms, 
-            Sports allows you to see the upcoming Sports games, and see live stats, player positions, and scores with TV Tag,
-            TV Tag allows you see the upcoming and completed programs that supported TV Tag, and you can interact with other people in these taglines, as seen in the previous
+            Nintendo TVii offers you the option to see your Favorites, TV shows airing live, or releasing new
+            episodes soon on Streaming platforms,
+            Movies also shows you the currently airing movies live, or the popular/trending ones on Streaming
+            platforms,
+            Sports allows you to see the upcoming Sports games, and see live stats, player positions, and scores

```

Comment:
are you using a formatter or linter? Make sure it doesn't appear like this so the result on the console isn't:

Going to be
like
this. Because that would be weird.

Reply:
it looks exactly the same as it did before